src/main/resources/api/api.yaml

```diff
@@ -250,6 +411,74 @@ definitions:
   #
   # Questions
   #
+  V1Question:
+    type: object
+    required:
+      - questionText
+      - questionType
+      - topic
+    properties:
+      id:
+        type: integer
+        format: int64
+      questionText:
+        type: string
+      questionType:
+        type: string
+        description: question type means the type of answer - multi, free, etc
+        default: many answers
+        enum:
+        - many answers
```

Comment:
fix enum to be compatible with QuestionType 
hm... we have no any names for question type in database...
I would like you to use ids here for now. For determining still use enum [1,2,3,4]
And add issue into project to anlarge question type and add unique type field for REST.